tests/test_grid_sampler.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import numpy as np
+import easyvvuq as uq
+from easyvvuq.actions import CreateRunDirectory, Encode, Decode, ExecuteLocal, Actions
+
+
+@pytest.fixture
+def sampler():
+    vary = {"x1": [0.0, 0.5, 1.0],
+            "x2": [True, False]}
+    return uq.sampling.Grid_Sampler(vary)
+
+
+@pytest.fixture
+def campaign():
+
+    params = {}
+    params["x1"] = {"type": "float", "default": 0.5}
+    params["x2"] = {"type": "boolean", "default": True}
+
+    # python file is its own template
+    encoder = uq.encoders.GenericEncoder('tests/grid_search/test_grid.py',
```

Comment:
perhaps change the name to 'tests/grid_search/test_grid.template' so that the invalid program 'tests/grid_search/test_grid.py' is not checked by the CI/CD system

Reply:
Right, I didn't know it would try to complile these. Will change to .template indeed